influxdb3_server/src/http/v1.rs

```diff
@@ -58,7 +58,8 @@ where
 
         let chunk_size = chunked.then(|| chunk_size.unwrap_or(DEFAULT_CHUNK_SIZE));
 
-        let stream = self.query_influxql_inner(database, &query).await?;
+        // TODO - params, currently passing None here:
```

Comment:
If this has an open issue mind putting it alongside the Todo?

Reply:
Good call-out. I opened https://github.com/influxdata/influxdb/issues/24805 and will update the comment.